assets/src/dashboard/components/typeahead-input/index.js

```diff
@@ -176,42 +214,55 @@ const TypeaheadInput = ({
       className={className}
       {...rest}
       isOpen={isMenuOpen}
+      isExpanded={isInputExpanded}
     >
       <InputContainer isOpen={isMenuOpen}>
-        <IconContainer>
-          <SearchIcon />
-        </IconContainer>
-        <label aria-label={ariaLabel} htmlFor={inputId} />
-        <StyledInput
-          autoComplete="off"
-          type="text"
-          id={inputId}
-          name={inputId}
-          disabled={disabled}
-          isOpen={isMenuOpen}
-          value={inputValue}
-          onFocus={() => setShowMenu(true)}
-          onChange={({ target }) => {
-            handleInputChange({ label: target.value, value: target.value });
+        <SearchButton
+          onClick={() => {
+            setMenuFocused(true);
+            inputRef.current.focus();
```

Comment:
Not a huge deal but might be worth putting in a `useCallback` so the fn doesn't get recreated.

Reply:
Good call out! I'll update this.